spec/game_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rspec'
+require_relative '../lib/req.rb'
+
+RSpec.describe Game do
+  let(:board) { Board.new('Eli', 'Satt') }
+  let(:g) { Game.new('Eli', 'X', 'Satt', 'O') }
+  let(:player1) { Player.new('Eli', 'X') }
+  let(:player2) { Player.new('Satt', 'O') }
+
+  describe '#check_winner' do
+    it 'winner_update must be false since the moves_counter is lesser than 5' do
+      expect(g.winner_update(player1, board)).to eq(false)
+    end
+  end
+
+  describe '#new_game?' do
+    it 'new_game must return true if user press [Q]' do
+      g.stub(:gets).and_return("q\n")
+      expect(g.new_game?).to eq(true)
+    end
+  end
+
+  describe '#new_game?' do
+    it 'new_game must return true if user press [Enter]' do
+      g.stub(:gets).and_return("\n")
+      expect(g.new_game?).to eq(false)
+    end
+  end
+
+  describe '#make_your_move?' do
+    it 'make_your_move must return value stored in @moves_counter' do
+      player1.stub(:gets).and_return("1\n")
+      expect(g.make_your_move(player1, board)).to eq(1)
+      player2.stub(:gets).and_return("2\n")
+      expect(g.make_your_move(player2, board)).to eq(2)
+      player1.stub(:gets).and_return("3\n")
+      expect(g.make_your_move(player1, board)).to eq(3)
+      player2.stub(:gets).and_return("4\n")
+      expect(g.make_your_move(player2, board)).to eq(4)
+      player1.stub(:gets).and_return("5\n")
+      expect(g.make_your_move(player1, board)).to eq(5)
+    end
+  end
+
+  describe '#winner_update' do
+    it '#winner_update must return false if there\'s no winner' do
+      expect(g.winner_update(player1, board)).to eq(false)
+    end
+  end
+
+  describe '#make_your_move?' do
+    it '#winner_update must return false if there is winner' do
+      player1.stub(:gets).and_return("1\n")
+      expect(g.make_your_move(player1, board)).to eq(1)
```

Comment:
same here it's very manual. every class is an object right, so instead of specifically saying player one, rather make an object that is going to behave like the player one. If player one is receiving the string for example and storing it, you can use that approach instead of going for `gets` method.